lix/packages/server/eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import lixStyle from "@lix-js/code-style"
+import globals from "globals"
+
+export default [
+	...lixStyle,
+	{
+		languageOptions: {
```

Comment:
add overrides for tests

Reply:
@martin-lysk  i saw we needed to ignore the node type issues for tests before too, so i will leave this until it becomes more annoying. lets try to keep node deps low anywyas, maybe make a helper module that has all the ignores or add this to the type setup